### PR TITLE
List countdown of business days until date

### PR DESCRIPTION
**Pull request by mmattozzi**
For Issue #7, provides an option to list the count of business days only until the target event date.

### CODE REVIEW

**Comment by famanson** [comment]
@Denvar94 hey Denvar, would you mind looking at this when you have some free time?

File: countdown.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import json
 import os
 import requests
+import numpy
```
Comment (famanson) [inline countdown.py]: I have not read the code thoroughly yet, but `numpy` seems like a bit overkill for the purpose right? :D

Comment (mmattozzi) [inline countdown.py]: I don't think it makes sense to do this calculation myself, but it appears there is another python library called [workdays](https://pypi.python.org/pypi/workdays/) that exists only for this purpose which I could use. Want me to switch?

Comment (famanson) [inline countdown.py]: @mmattozzi yes please if you don't mind!

Comment (mmattozzi) [inline countdown.py]: Done... switched to using `workdays`

File: countdown.py [hunk 2]
```diff
@@ -34,47 +35,56 @@ def days_from_christmas():
...
-    return delta.days + 1
+    if business_days:
+        delta = workdays.networkdays(currentdate, futuredate)
+    else:
+        delta = (futuredate - currentdate).days + 1
+    return delta
 
     
-def events(strdate,event):
+def events(strdate,event,business_days):
```
Comment (famanson) [inline countdown.py]: @mmattozzi just one more minor cosmetics thing before I merge this. It seems like in some places we're missing spaces after some of the commas (our fault initially). Would you mind changing that too?

Comment (mmattozzi) [inline countdown.py]: Added spaces.

**Comment by famanson** [comment]
@mmattozzi I just merged this! Thanks for contributing :D

**Pull request merged**